enhancements/installer/rhel8_worker_support.md

```diff
@@ -0,0 +1,252 @@
+the same Ansible playbook process however, RPM content will be delivered using
+an image in the release package.
+
+## Motivation
+
+Customers desire to deploy RHEL 8 nodes and using containerized content will provide
+a better user experience than configuring RPM repositories on nodes.  Using the
+existing process will allow near-term support of RHEL 8 nodes while enhanced
+longer-term RHEL 8 support is developed.
+
+### Goals
+
+1. Implement an os-content image for RHEL 8 RPMs
+1. Update [openshift-ansible] scaleup.yml and upgrade.yml playbooks to support RHEL 8 workers
+
+### Non-Goals
+
+* Deploying/upgrading RHEL 8 worker nodes through other mechanisms, i.e. operators
+* Changing the deployment/upgrade process for RHEL 7 worker nodes
+* Upgrading existing RHEL 7 worker nodes to RHEL 8
+
+## Proposal
+
+Currently, for RHEL 7 worker node deployment, the user must [prepare the host][prepare-rhel]
+before running the scaleup.yml playbook.  This process can be cumbersome for
+automating deployment and also requires additional steps to update repos prior
+to performing an upgrade.  For RHEL 8 hosts preparing repos on the hosts will
+not be required because the necessary packages will be delivered as an image in
+the release package.  This removes the need of mirroring the release package as
+well as setting up repo mirrors for offline or disconnected deployments.
+
+The basic process for scaling up and upgrading RHEL worker nodes will remain the
+same.  The playbooks will be updated to download and mount the image with the RPM
+content during the install/upgrade for RHEL 8 hosts.  The install process will
+not change for RHEL 7 worker nodes.
+
+[prepare-rhel]: https://docs.openshift.com/container-platform/4.7/machine_management/adding-rhel-compute.html#rhel-preparing-node_adding-rhel-compute
+
+### User Stories
+
+1. As an OpenShift admin using [openshift-ansible], I want to deploy and upgrade
+   RHEL 8 Server worker nodes without needing to prepare hosts for installation.
+
+1. As an Openshift admin, I want to be able to deploy RHEL 8 workers without
+   having to mirror RPM content.
+
+### Implementation Details/Notes/Constraints [optional]
+
+What are the caveats to the implementation? What are some important details that
+didn't come across above. Go in to as much detail as necessary here. This might
+be a good place to talk about core concepts and how they relate.
+
+### Risks and Mitigations
+
+What are the risks of this proposal and how do we mitigate. Think broadly. For
+example, consider both security and how this will impact the larger OKD
+ecosystem.
+
+How will security be reviewed and by whom? How will UX be reviewed and by whom?
+
+Consider including folks that also work outside your immediate sub-project.
+
+## Design Details
+
+### Open Questions [optional]
+
+1. How will this be implemented for OKD?  Is this a concern?
+1. Can we use existing os-content used for RHCOS to install required RPMs?
```

Comment:
```suggestion
1. Can we use existing os-content used for RHCOS to install required RPMs or files?
```
Given the discussion during the arch call where it was mentioned perhaps we just extract raw files out of the os-content image.